pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -75,8 +75,17 @@ type PostgresClusterSpec struct {
 	// +optional
 	CustomReplicationClientTLSSecret *corev1.SecretProjection `json:"customReplicationTLSSecret,omitempty"`
 
-	// The image name to use for PostgreSQL containers
-	// +kubebuilder:validation:Required
+	// The image name to use for PostgreSQL containers. The image may also be set
+	// using an environment variable. For standard PostgreSQL images, the format
+	// is RELATED_IMAGE_POSTGRES_<Version> where Version is the PostgreSQL
+	// version set in PostgresVersion.
+	// Example: RELATED_IMAGE_POSTGRES_13
+	// For PostGIS enabled PostgreSQL images, the format is
+	// RELATED_IMAGE_POSTGRES_<Version>_GIS_<GIS Version> where "Version" is the
+	// PostgreSQL version set in PostgresVersion and "GIS Version" is the PostGIS
+	// version set in PostGISVersion.
+	// Example: RELATED_IMAGE_POSTGRES_13_GIS_3.1
```

Comment:
📝 Line breaks in these comments are removed when added to the CRD, and it looks like `kubectl explain` might ... unwrap (?) even more. `Example: RELATED_IMAGE_POSTGRES_13` bleeds into the following sentence making it a little confusing.

🌱 Seeing it now, I'm not sure I like talking about operator environment variables in the CRD spec. The person deploying PostgresClusters may not know anything about how the operator was deployed. A section about configuring/installing the operator in our [Markdown] documentation seems like the right way to go, but it's not necessary for this PR.

🔧 We should refer to other fields by their JSON/YAML names, like `postgresVersion` rather than `PostgresVersion`. Is it plausible to use those names when describing the env var format?

```suggestion
	// The image name to use for PostgreSQL containers. When omitted, the value
	// comes from an operator environment variable. For standard PostgreSQL images,
	// the format is RELATED_IMAGE_POSTGRES_<postgresVersion>,
	// e.g. RELATED_IMAGE_POSTGRES_13. For PostGIS enabled PostgreSQL images,
	// the format is RELATED_IMAGE_POSTGRES_<postgresVersion>_GIS_<postGISVersion>,
	// e.g. RELATED_IMAGE_POSTGRES_13_GIS_3.1.
```

Reply:
The suggestion looks good to me, so I've updated to that. 

As for the question regarding the documentation/use of the JSON/YAML names, for the time being I think the suggested update is sufficient. It's clear which variables are used to update the environment variable names, and we have corresponding examples in `config/manager/manager.yaml` that should make the usage fairly straightforward.